### PR TITLE
Upgrade py dependency version for wazuh-qa when Python is in version 3.11

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|-------------|
|https://github.com/wazuh/wazuh-jenkins/issues/5407|

## Description

When testing the changes in PR https://github.com/wazuh/wazuh-jenkins/pull/5342, it has been seen the tests fail with an error shown later if the dependency `py` is in version `1.10` instead of `1.11`, as seen in https://github.com/pytest-dev/apipkg/issues/30.

### Updated

- requirements.txt
---

## Testing performed

The test fails on some fim tests, but it is expected, as running it using branch 4.5.3 also fails with it. 

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @user (Developer)  |           | [:green_circle: ](https://ci.wazuh.info/job/Test_integration/43006/) | ⚫⚫⚫  |         |         | Nothing to highlight |
| @user (Reviewer)   |           | ⚫⚫⚫ | :no_entry_sign: :no_entry_sign: :no_entry_sign:  |        |         | Nothing to highlight |

### CODE REVIEW

**Comment by verdx** [comment]
The rest of the updates of this PR are reflected on issue https://github.com/wazuh/wazuh-jenkins/issues/5407, as there are two different PRs and the updates are common.

The testing on why this PR can be merged can be found in the last commentary in the issue.

**Review by damarisg — CHANGES_REQUESTED** [review]
Please add the CHANGELOG file

**Comment by verdx** [comment]
> Please add the CHANGELOG file

@damarisg Done in https://github.com/wazuh/wazuh-qa/pull/4523/commits/7780af6af43a1bfd7923e1e6df9bbab729ea522a

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 Wazuh commit: TBD \
 Release report: TBD
 
+## Fixed
```
Comment (juliamagan) [inline CHANGELOG.md:10]: This entry is duplicated

**Review by juliamagan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (verdx) [reply CHANGELOG.md]: Fixed in https://github.com/wazuh/wazuh-qa/pull/4523/commits/d599dd8db741eae95fd4d6d758c155a45edd98cb

**Pull request merged**